packages/docs/src/locales/en/en.json

```diff
@@ -758,7 +758,8 @@
       },
       "events": {
```

Comment:
When i focus select and i press tab - I expect next select to be focused.
Instead focus is lost.

Reply:
I have this behavior when press `tab`
![chrome-capture](https://user-images.githubusercontent.com/55198465/145564310-70da30c6-6e3d-4ba7-98a6-ee50b8478562.gif)